code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -93,13 +91,14 @@
 	if(health < HEALTH_THRESHOLD_DEAD && src.stat != 2) //die only once
 		death()
 
+
 	if (src.stat != 2) //Alive.
-		if (src.paralysis || src.stunned || src.weakened || !src.has_power) //Stunned etc.
+		if (src.paralysis || src.stunned || !src.has_power || src.weakened) //Stunned etc.
 			src.stat = 1
-			if (src.stunned > 0)
-				AdjustStunned(-1)
 			if (src.weakened > 0)
 				AdjustWeakened(-1)
+			if (src.stunned > 0)
+				AdjustStunned(-1)
```

Comment:
Why?

Reply:
i did the entire commit in one big piece so i had to manually rewrite the code to act like the original

Reply:
By rewriting it exactly the way it was before, it would have not shown changes.